Configuration/Realm/Realm.xcconfig

```diff
@@ -27,11 +27,11 @@ LD_RUNPATH_SEARCH_PATHS[sdk=iphone*] = $(inherited) @executable_path/Frameworks
 LD_RUNPATH_SEARCH_PATHS[sdk=watch*] = $(inherited) @executable_path/Frameworks @loader_path/Frameworks;
 LD_RUNPATH_SEARCH_PATHS[sdk=appletv*] = $(inherited) @executable_path/Frameworks @loader_path/Frameworks;
 
-OTHER_LDFLAGS[sdk=macosx*] = -lrealm$(REALM_LIBRARY_SUFFIX);
+OTHER_LDFLAGS[sdk=macosx*] = -lrealm$(REALM_LIBRARY_SUFFIX) -framework SystemConfiguration;
```

Comment:
Related to dynamic loading: according to docs `dlopen` seems to work on iOS8+ while we still support iOS7. I didn't find if it works for system frameworks though. I've also noticed some SO question saying `dlopen` is not supported in app extensions but didn't find any proofs in docs.
Do you know something about that? As a second option we can mention `SystemConfiguration` in docs the same way as `libc++`.